python/sbp/utils.py

```
@@ -78,7 +78,8 @@ def fmt_repr(obj):
 # TODO: Once https://github.com/construct/construct/pull/59 is published
 #       to pypi, remove this in favor of construct.GreedyString
 def greedy_string(name):
```

I would recommend having an assertion here as opposed to failing silently.